kolibri/plugins/html5_viewer/assets/src/views/Html5AppRendererIndex.vue

```diff
@@ -114,7 +123,8 @@
 
   .html5-renderer {
     position: relative;
-    height: 500px;
+    min-height: 500px;
+    max-height: 70vh;
```

Comment:
`vh` has [mediocre support](https://caniuse.com/#feat=viewport-units) on older iOS and Android devices.

We can do this, but if there is a way to avoid `vh` that would be preferable.

Reply:
Any ideas on alternatives? Doing percentage would make it relative to parent element, rather than expanding the height based on available screen space, wouldn't it?

Reply:
I'll take a look when I actually test it locally